lib/rules/web/admin_console/4.7/status.xyaml

```diff
@@ -50,4 +50,4 @@ check_name_validation_error:
   - selector:
       xpath: //h4[contains(.,'error occur')]
   - selector:
-      xpath: //div[contains(.,'subdomain must consist of lower case alphanumeric character')]
+      xpath: //div[contains(text(),'subdomain must consist of lower case alphanumeric character')]
```

Comment:
Could you help explain `The error message will disappear on page when too many results found on fuzzy query` ? I didn't get your point